py-polars/polars/convert.py

```diff
@@ -588,9 +588,14 @@ def from_arrow(
         3
     ]
     """  # noqa: W505
-    if isinstance(data, pa.Table):
+    if isinstance(data, pa.RecordBatch):
+        data = [data]
```

Comment:
Can we move this to below the check for `Array`? I recon `Table` is most common, then array, then recordbatch. That saves a few checks in the more common cases.

Reply:
Gone one better and trivially improved `_from_arrow`; can properly handle both `Table` _and_ `RecordBatch` input now, so we only need the one outer `isinstance` check 👍